### PR TITLE
prepare

### PR DESCRIPTION
**Pull request by superyuuki**

### CODE REVIEW

File: RapidReact/src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -198,5 +197,15 @@ private void configureButtonBindings() {
     buttons.outtake.whenPressed(intakeSubsystem::spinBackward);
     buttons.intake.whenReleased(intakeSubsystem::stopSpin);
     buttons.outtake.whenReleased(intakeSubsystem::stopSpin);
+
+    //Shooter BUttons
+    buttons.hubShoot.whenPressed(() -> shooterSubsystem.shootTop());
+    buttons.hubShoot.whenReleased(shooterSubsystem::stopShootTop);
+    buttons.lowShoot.whenPressed(() -> {
```
Comment (cpostbitbuckets) [inline RapidReact/src/main/java/frc/robot/Robot.java:204]: low shoot button should low shoot. The tarmacShoot button should tarmacShoot.

File: RapidReact/src/main/java/frc/robot/subsystem/ShooterSubsystem.java [hunk 2]
```diff
@@ -1,13 +1,34 @@
...
+  }
+
+  public void shootTarmac() {
+
+  }
+
+  public void stopShootTarmac() {
+
+  }
+
```
Comment (cpostbitbuckets) [inline RapidReact/src/main/java/frc/robot/subsystem/ShooterSubsystem.java:31]: can we add logging to these so we know they are turning on/off?

File: RapidReact/src/main/java/frc/robot/subsystem/ShooterSubsystem.java [hunk 3]
```diff
@@ -1,13 +1,34 @@
 package frc.robot.subsystem;
 
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
 import frc.robot.config.Config;
 
 public class ShooterSubsystem extends BitBucketsSubsystem {
 
+  private TalonSRX doSomethingElseIdk;
+  private TalonSRX launchTheBalls;
```
Comment (cpostbitbuckets) [inline RapidReact/src/main/java/frc/robot/subsystem/ShooterSubsystem.java:9]: The shooter is actually using REV motors I think, and we should name them something that makes sense, like shooterMotor and feederMotor.

**Review by cpostbitbuckets — CHANGES_REQUESTED** [review]

File: RapidReact/src/main/java/frc/robot/Robot.java [hunk 4]
```diff
@@ -198,5 +197,19 @@ private void configureButtonBindings() {
     buttons.outtake.whenPressed(intakeSubsystem::spinBackward);
     buttons.intake.whenReleased(intakeSubsystem::stopSpin);
     buttons.outtake.whenReleased(intakeSubsystem::stopSpin);
+
+    //Shooter BUttons
+    buttons.hubShoot.whenPressed(shooterSubsystem::shootTop);
+    buttons.hubShoot.whenReleased(shooterSubsystem::stopShootTop);
+
+    buttons.lowShoot.whenPressed(shooterSubsystem::shootLow);
+    buttons.lowShoot.whenReleased(shooterSubsystem::stopShootLow);
```
Comment (cpostbitbuckets) [inline RapidReact/src/main/java/frc/robot/Robot.java:206]: I don't think we need separate stop* functions. Stop should just be "turn off the motors"

Comment (superyuuki) [reply RapidReact/src/main/java/frc/robot/Robot.java]: done

**Review by cpostbitbuckets — APPROVED** [review]

**Pull request merged**